server/planning/feed_parsers/superdesk_event_json.py

```diff
@@ -0,0 +1,171 @@
+import logging
+import datetime
+
+from superdesk.io.feed_parsers import FileFeedParser
+import pytz
+import json
+
+utc = pytz.UTC
+logger = logging.getLogger(__name__)
+
+
+class SuperdeskFeedParser(FileFeedParser):
```

Comment:
A more descriptive name here would define what this feed parser is for, such as `EventJsonFeedParser`